iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractLineTest.java

```diff
@@ -436,7 +436,7 @@ public void testRemoveAcLine() {
         Line line = network.getLine(TO_REMOVE);
         assertNotNull(line);
         int count = network.getLineCount();
-        line.remove();
+        line.remove();    // <---- TODO HERE
```

Comment:
On network-store-store's side : line.getTerminal1() works but then t1.isConnected() raise exception with message "Object has been removed in current variant" because the resource of the terminal instance is null